globalvars.py

```diff
@@ -20,194 +20,210 @@ class GlobalVars:
+            "280934",   # M.A.Ramezani
+            "200235",   # durron597
+            "148310",   # Awesome Poodles / Brock Adams
+            "168333",   # S.L. Barth
+            "257207",   # Unikitty
+            "244282",   # DroidDev
+            "163250",   # Cupcake
+            "298265",   # BoomsPlus
+            "253560",   # josilber
+            "244254",   # misterManSam
+            "188189",   # Robert Longson
+            "174699",   # Ilmari Karonen
+            "202362",   # chmod 666 telkitty
+            "289717",   # Quill
+            "237813",   # bjb568
+            "311345",   # Simon Klaver
+            "171881",   # rekire
+            "260388",   # Pandya
+            "310756",   # Ms Yvette
+            "262399",   # Jeffrey Bosboom
+            "242209",   # JAL
+            "280883",   # ByteCommander
+            "302251",   # kos
+            "262823"    # ArtOfCode
+        ],
+        socvr_room_id: [
+            "1849664",  # Undo
+            "2581872",  # hichris123
+            "1198729",  # Manishearth
+            "3717023",  # Normal Human aka 1999
+            "2619912",  # ProgramFOX
+            "578411",   # rene
+            "1043380",  # gunr2171
+            "2246344",  # Sam
+            "2756409",  # TylerH
+            "1768232",  # durron597
+            "359284",   # Kevin Brown
+            "258400",   # easwee
+            "3622940",  # Unihedron
+            "3204551",  # Deduplicator
+            "4342498",  # NathanOliver
+            "4639281",  # Tiny Giant
+            "3093387",  # josilber
+            "1652962",  # cimmanon
+            "1677912",  # Mogsdad
+            "656243",   # Lynn Crumbling
+            "3933332",  # Rizier123
+            "2422013",  # cybermonkey
+            "3478852",  # Nisse Engström
+            "2302862",  # Siguza
+            "1324",     # Paul Roub
+            "1743880",  # Tunaki
+            "1663001",  # DavidG
+            "2415822",  # JAL
+            "4174897",  # Kyll
+            "5299236",  # Kevin Guan
+            "4050842",  # Thaillie
+            "1816093",  # Drew
+            "874188",   # Triplee
+            "880772",   # approxiblue
+            "1835379",  # Cerbrus
+            "1843331",  # Tim Castelijns
+            "3956566",  # JamesENL
+            "2357233",  # Ms Yvette
+            "3155639",  # AlexanderOMara
+            "462627",   # Praveen Kumar
+            "4490559",  # intboolstring
+            "1364007",  # Wai Ha Lee
+            "1699210",  # bummi
+            "563532",   # Rob
+            "5389107",  # Magisch
+            "4099593",  # bhargav-rao
+            "1542723",  # Ferrybig
+            "2025923",  # Tushar
+            "5292302",  # Petter Friberg
+            "792066",   # Braiam
+            "5666987",  # Ian
+            "3160466",  # ArtOfCode
+            "5735775",  # Ashish Ahuja
+            "3476191"   # Nobody Nada
+        ]
+    }
+    smokeDetector_user_id = {
```

Comment:
Oh, no, I meant https://github.com/Charcoal-SE/SmokeDetector/pull/159#discussion_r64142822 (making `smokeDetector_user_id` and the ones below one-liners)